### PR TITLE
ODS-5476 Move ODS/API: Promote Packages build to Github Actions

### PR DESCRIPTION
**Pull request by semalaiappan**

### CODE REVIEW

File: .github/workflows/Promote Packages.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+name: Promote Packages
+
+on:
+  workflow_dispatch:
+    inputs:
+      promote_packages_view:
+        description: ''select promote packages view for Azure Artifacts'
```
Comment (bptillman) [inline .github/workflows/Promote Packages.yml:12]: There is an extra single quote at the start of this description that makes this yml file invalid, so it needs to be removed.

File: .github/workflows/Promote Packages.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        required: true
+        default: 'Prerelease'
+        type: choice
+        options:
+        - Prerelease
+        - Release
+        - Local
```
Comment (bptillman) [inline .github/workflows/Promote Packages.yml:19]: If we need to keep the Local option for testing then that is fine. But I tested in my own fork of the repo and only two options are required for the input to ensure its a drop down, as seen here 
image

**Review by bptillman — CHANGES_REQUESTED** [review]
Just one required change to make here and a second optional one, depending on if it needs to stay around.

File: .github/workflows/Promote Packages.yml (hunk 1 above)
Comment (semalaiappan) [reply .github/workflows/Promote Packages.yml]: I removed it

File: .github/workflows/Promote Packages.yml (hunk 2 above)
Comment (semalaiappan) [reply .github/workflows/Promote Packages.yml]: I removed local

**Review by bptillman — APPROVED** [review]

**Pull request merged**